restAssuredCatFacts/src/main/java/api/AppClient.java

```diff
@@ -1,22 +1,22 @@
-package api;
-
-import io.restassured.http.ContentType;
-import io.restassured.http.Method;
-import io.restassured.response.Response;
-import io.restassured.specification.RequestSpecification;
-
-public class AppClient {
-
-    private static final String BASE_URI = "https://catfact.ninja/";
-    public Response factsResponse(Method method, RequestSpecification requestSpec) {
-        requestSpec.baseUri(BASE_URI)
-            .contentType(ContentType.JSON);
-        return requestSpec.request(method);
-    }
-
-    public Response breedsResponse(Method method, RequestSpecification requestSpec) {
-        requestSpec.baseUri(BASE_URI)
-            .contentType(ContentType.JSON);
-        return requestSpec.request(method);
-    }
+package api;
+
+import io.restassured.http.ContentType;
+import io.restassured.http.Method;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+public class AppClient {
+
+    private static final String BASE_URI = "https://catfact.ninja/";

```

Comment:
Previously we added the aproach with PropertyReader and in this peace of code it can be realy useful. In future please don't forget to implement improvemets from previouse tasks